podcasts/Onboarding/Plus/UpgradeLandingView.swift

```diff
@@ -147,7 +146,7 @@ struct UpgradeTier: Identifiable {
 
 extension UpgradeTier {
     static var plus: UpgradeTier {
-        UpgradeTier(tier: .plus, iconName: "plusGold", title: "Plus", yearlyIdentifier: Constants.IapProducts.yearly.rawValue, monthlyIdentifier: Constants.IapProducts.monthly.rawValue, description: L10n.accountDetailsPlusTitle, buttonLabel: L10n.plusSubscribeTo, buttonColor: Color(hex: "FFD846"), buttonForegroundColor: Color.plusButtonFilledTextColor, features: [
+        UpgradeTier(tier: .plus, iconName: "plusGold", title: "Plus", plan: .plus, description: L10n.accountDetailsPlusTitle, buttonLabel: L10n.plusSubscribeTo, buttonColor: Color(hex: "FFD846"), buttonForegroundColor: Color.plusButtonFilledTextColor, features: [
```

Comment:
Should we use the L10n.pocketCastsPlusShort here?

Reply:
Good question. Plus is not translated at all in any language

Reply:
True, my thinking was more keeping the wording in a single place rather than translating it.